### PR TITLE
replaces react-quill with react-quill-new

### PR DESCRIPTION
**Pull request by tillyw**
## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/17450

this is a holdover to replace the react-quill library with a patched version that is compatible with upcoming Chrome updates

we will remove this library once we have [lexical-react](https://www.npmjs.com/package/@lexical/react) in place as our new rich text editor

## Testing
**URL to test:** 

https://deploy-preview-1375--atd-moped-main.netlify.app/

**Steps to test:**
- open a project notes tab and ensure that the rich text editor is working

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Product manager checked [DB view dependencies](https://atd-dts.gitbook.io/moped-documentation/product-management/updating-the-arcgis-online-components-database-view)
- [ ] Product manager added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-editor/src/views/projects/projectView/NoteInputQuill.js [hunk 1]
```diff
@@ -9,8 +9,8 @@ import {
   RadioGroup,
   FormControlLabel,
 } from "@mui/material";
-import ReactQuill from "react-quill";
-import "react-quill/dist/quill.snow.css";
+import ReactQuill from "react-quill-new";
+import "react-quill-new/dist/quill.snow.css";
```
Comment (mddilley) [inline moped-editor/src/views/projects/projectView/NoteInputQuill.js:13]: I see one compile warning related to this theme import, but it won't be there long so I'm not concerned about it. 👍

**Review by mddilley — APPROVED** [review]
I tested every place that this editor is used, and I see no regressions. Thanks for putting this in place while you continue to work on the replacement library. 🙏 🚀

**Comment by mddilley** [comment]
also, if anyone was curious, the diff between this fork and the library forked from you can see it [here](https://github.com/zenoamaro/react-quill/compare/master...VaguelySerious:react-quill:master). I feel good that we invested time in a new solution for this feature, and I'm glad that someone put in the work into this fork to hold us over. 💯

**Review by chiaberry — APPROVED** [review]
thank you!

**Pull request merged**